app/javascript/packs/application.js

```diff
@@ -27,10 +27,12 @@ import "bootstrap";
 
 // Internal imports, e.g:
 import { initModal } from '../plugins/init_modal';
+import { initLocation } from '../plugins/init_location';
 
 document.addEventListener('turbolinks:load', () => {
   // Call your functions here, e.g:
   initModal();
+  initLocation(); // only work in the home page
```

Comment:
Do you mean like only needs to work in the home page?

Reply:
Remove comment pls

Reply:
removed